features/support/env.rb

```diff
@@ -1,3 +1,6 @@
+require 'coveralls'
```

Comment:
You've updated this file in all your PRs? You should probably remove this from this PR and merge the other ones that include this change ASAP to avoid having to set this up over and over again

Reply:
Will let everyone know

Reply:
Done